source/guide_gitea.rst

```diff
@@ -29,62 +28,124 @@ of the then-popular Gogs_, but quickly became more active and bigger than the or
 
-  100%[=======================================================>] 52,960,072  9.99MB/s   in 5.8s
+.. code-block:: console
+
+  [isabell@stardust ~]$ gpg --keyserver keys.gnupg.net --recv-keys 7C9E68152594688862D62AF62D9AE806EC1592E2
+  gpg: directory `/home/isabell/.gnupg' created
+  gpg: new configuration file `/home/isabell/.gnupg/gpg.conf' created
+  gpg: WARNING: options in `/home/isabell/.gnupg/gpg.conf' are not yet active during this run
+  gpg: keyring `/home/isabell/.gnupg/secring.gpg' created
+  gpg: keyring `/home/isabell/.gnupg/pubring.gpg' created
+  gpg: requesting key EC1592E2 from hkp server keys.gnupg.net
+  gpg: /home/isabell/.gnupg/trustdb.gpg: trustdb created
+  gpg: key EC1592E2: public key "Teabot <teabot@gitea.io>" imported
+  gpg: no ultimately trusted keys found
+  gpg: Total number processed: 1
+  gpg:               imported: 1  (RSA: 1)
+  [isabell@stardust ~]$ gpg --verify ~/gitea/gitea.asc ~/gitea/gitea
+  gpg: Signature made Sat 09 May 2020 10:19:06 PM CEST using RSA key ID 9753F4B0
+  gpg: Good signature from "Teabot <teabot@gitea.io>"
+  gpg: WARNING: This key is not certified with a trusted signature!
+  gpg:          There is no indication that the signature belongs to the owner.
+  Primary key fingerprint: 7C9E 6815 2594 6888 62D6  2AF6 2D9A E806 EC15 92E2
+       Subkey fingerprint: CC64 B1DB 67AB BEEC AB24  B645 5FC3 4632 9753 F4B0
+  [isabell@stardust ~]$ 
+
+If the verification is fine, we get a ``gpg: Good signature from "Teabot <teabot@gitea.io>"`` line and we make the bin executeable.
+  
+.. code-block:: console
 
-  2018-09-16 18:36:36 (8.72 MB/s) - gitea/gitea’ saved [60473144/60473144]
-  [isabell@stardust ~]$ sha256sum gitea/gitea
-  b84098f0b0018071aa1ba5522078820494def29e6385c25c581c4362a4e463b3 gitea/gitea
-  [isabell@stardust ~]$ chmod +x gitea/gitea
+  [isabell@stardust ~]$ chmod u+x ~/gitea/gitea
   [isabell@stardust ~]$
 
+
 Configuration
 =============
 
-Change the configuration
-------------------------
+Gitea configuration file
+-------------------------
 
-You need to create a custom directory
+Before we write the configuration we need the MySQL database password and some random characters as security key. We catch the MySQL/MariaDB password ...
 
-::
+.. code-block:: console
 
-  [isabell@stardust ~]$ mkdir -p ~/gitea/custom/conf/
-  [isabell@stardust ~]$
+  [isabell@stardust ~]$ my_print_defaults --defaults-file=~/.my.cnf client | grep -- --password | awk --field-separator = '{ print $2 }'
+  xG9MD-15A4aMGsln,MTr
+  [isabell@stardust ~]$ 
+
+... and some random alpha-num chars for the security key. For this we pipe some random characters from the ``/dev/urandom`` device into the translate tool ``tr`` to delete the complementary of alpha-numeric characters until we have 32 characters.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cat /dev/urandom | tr --delete --complement 'a-zA-Z0-9' | fold --width=32 | head --lines=1
```

Comment:
newer versions of U7 offer "pwgen", which enables your to shorten this line to `pwgen 32`.

Reply:
Done.